confluent_platform/changelog.d/17490.fixed

```diff
@@ -0,0 +1,2 @@
+
+Fix correct Mbean regex stream thread metric(after version 5.5.0 of confluent platform). Previously, because Bean Regex was set to Client metrics, Stream Thread Metrics were not collected. also added one useful attribute for debugging Stream Thread performance.
```

Comment:
```suggestion
Add a Bean regex for stream thread metrics for Confluent version >=5.5.0. Previously, Stream Thread Metrics were not collected. Also added one useful attribute for debugging stream thread performance.
```

Reply:
Thanks I think your suggestion is more appropriate.